math/libtommath/files/patch-makefile.diff

```diff
@@ -0,0 +1,18 @@
+--- makefile.orig	2016-02-03 18:07:27.000000000 +0000
++++ makefile	2017-07-14 14:39:19.000000000 +0000
+@@ -53,10 +53,15 @@
+ 
+ #END_INS
+ 
++all: libtommath.$(VERSION).dylib $(LIBNAME)
++
+ $(LIBNAME):  $(OBJECTS)
+ 	$(AR) $(ARFLAGS) $@ $(OBJECTS)
+ 	$(RANLIB) $@
+ 
++libtommath.$(VERSION).dylib: $(OBJECTS)
++	$(CC) -fno-common -dynamiclib -o $@ $(OBJECTS) -install_name $(LIBPATH)/$@
```

Comment:
I think you should set `-current_version $(VERSION) -compatibility_version $(VERSION)` or something similar and set the ID load command using the `-install_name` parameter to the path of the library.

Can you also run `sudo port rev-upgrade --id-loadcmd-check` after installing this port to check whether the ID load command is set correctly.